### ISSUE TITLE
Module ‘Network.Wai.Handler.Warp’ does not export ‘settingsPort’

### ISSUE THREAD

**Issue by amitaibu** [snoyberg/haskellers#34]
I'm trying to follow the instructions on how to build the project locally. On the last step of `stack exec -- yesod devel` I getting:

```
devel.hs:4:36:
    Module ‘Network.Wai.Handler.Warp’ does not export ‘settingsPort’
```

**Comment by amitaibu** [comment]
(PR will follow)

**Comment by amitaibu** [comment]
Closed in favor of #35
